bot/handlers/user/subscription/payment_methods.py

```
@@ -101,41 +102,41 @@ async def payment_method_bind(callback: types.CallbackQuery, settings: Settings,
+    except Exception as exc:
+        logging.debug("Suppressed exception in bot/handlers/user/subscription/payment_methods.py: %s", exc)
 
 
 @router.callback_query(F.data.startswith("pm:delete_confirm"))
 async def payment_method_delete_confirm(callback: types.CallbackQuery, settings: Settings, i18n_data: dict):
     current_lang = i18n_data.get("current_language", settings.DEFAULT_LANGUAGE)
     i18n: Optional[JsonI18n] = i18n_data.get("i18n_instance")
-    if not getattr(settings, 'YOOKASSA_AUTOPAYMENTS_ENABLED', False):
+    if not settings.yookassa_autopayments_active:
         try:
             _ = lambda key, **kwargs: i18n.gettext(current_lang, key, **kwargs) if i18n else key
```

Ruff (0.15.1)

[error] Do not assign a `lambda` expression, use a `def`

Rewrite `_` as a `def`

(E731)